src/components/shows/heroBanner.module.css

```diff
@@ -0,0 +1,75 @@
+.hero {
+  color: white;
+  background-image: linear-gradient(
+      to right,
+      rgba(0, 0, 0, 0),
+      rgba(0, 0, 0, 0.75) 50%,
+      rgba(0, 0, 0, 0)
+    ),
+    url(../../assets/images/watch/curtains.jpg); /* Still stored in previous. */
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-attachment: fixed;
+  min-height: 100px;
+  height: 20vh;
+}
+
+.heroContainer {
+  margin: 0 auto;
+  width: 100%;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.heroTextArea {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  text-align: center;
+}
+
+.subtitle {
+  text-align: center;
+  font-size: 20px;
+  display: block;
+  font-style: italic;
+}
+
+.title {
+  font-size: 50px;
+  text-align: center;
+  margin-bottom: 3px;
+}
+
+@media only screen and (max-width: 768px) {
+  .subtitle {
+    text-align: center;
+    font-size: 14px;
+    display: block;
+    font-style: italic;
+  }
+
+  .title {
+    font-size: 38px;
+    text-align: center;
+    margin-bottom: 3px;
+  }
+}
+
+@media only screen and (max-width: 425px) {
```

Comment:
These font sizes are too small. Text should never really get smaller than the body size (16px) unless in exceptional circumstances as it's just too difficult to read.
In general, your heading and body text sizes should be more like the second image. Don't be afraid to use more vertical space, smartphones are tall.

![image](https://user-images.githubusercontent.com/1442037/79943487-1c7f1f80-84bd-11ea-9246-415691248058.png)